editor/editor_inspector.cpp

```diff
@@ -1291,6 +1329,13 @@ EditorInspectorSection::EditorInspectorSection() {
 	foldable = false;
 	vbox = memnew(VBoxContainer);
 	vbox_added = false;
+
+	dropping = false;
+	dropping_unfold_timer = memnew(Timer);
+	dropping_unfold_timer->set_wait_time(EDITOR_GET("interface/inspector/drag_and_drop_section_expand_delay"));
```

Comment:
It would be better to avoid bloating the bloat settings for that kind of things. Please, simply choose a value that seem fine to you and it should be ok for everyone. (If it's not, let's make it configurable, but I highly doubt we will have any complain)

Reply:
I found 0.6 seconds to be a good value, which is what the default was set to. This time is also show in the videos.

Requested changes have been implemented.

Reply:
That sounds good to me :)